lib/utils/messages.ts

```diff
@@ -1,10 +1,11 @@
 const DEBUG_MODE = !!process.env.DEBUG;
 
 export const reachingOut = `\
-If you need help, or just want to say hi, don't hesitate in reaching out at:
+If you need help, or just want to say hi, don't hesitate in reaching out
+through our discussion and support forums at https://forums.balena.io
 
-  Forums: https://forums.balena.io
-  GitHub: https://github.com/balena-io/balena-cli/issues/new\
+For bug reports or feature requests, have a look at the GitHub issues or
+create a new one at: https://github.com/balena-io/balena-cli/issues/\
```

Comment:
This change was not requested the deprecation issue... Simply coincidental. What happens is that some users create Github issues for support requests. For a couple of recent examples:
* [version mismatch of Balena CLI](https://github.com/balena-io/balena-cli/issues/1144)
* [balena join failed](https://github.com/balena-io/balena-cli/issues/1151)

The first one above was even said to be "urgent" by the user, but the forums have better chance of a quicker response. Sometimes users may believe that the issue is actually a bug, nonetheless I thought there was a case to make the CLI messages clearer.